deepspeed/runtime/engine.py

```diff
@@ -1616,6 +1700,17 @@ def deepspeed_io(self,
             data_parallel_world_size = self.mpu.get_data_parallel_world_size()
             data_parallel_rank = self.mpu.get_data_parallel_rank()
 
+        deepspeed_engine_config = {}
+        if self.curriculum_learning_enabled():
+            deepspeed_engine_config = {
+                "curriculum_learning_enabled": self.curriculum_learning_enabled(),
```

Comment:
Use symbolic constants instead of hard coded. Okay, to add as TODO.

Reply:
Have changed them to symbolic constants.